vocabularies/UI.xml

```diff
@@ -1389,6 +1390,55 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated entity type can be visualized in a reuse component for notes" />
```

Comment:
I wouldn't write something about reuse components - the term definition should be more timeless - it's defined for the visualization of notes. What do the others think?